Sources/SwiftCrossUI/Views/Modifiers/EnvironmentModifier.swift

```diff
@@ -37,3 +37,21 @@ package struct EnvironmentModifier<Child: View>: View {
         )
     }
 }
+
+extension View {
+    /// Modifies the environment of the View its applied to.
+    public func environment<T: EnvironmentKey>(key: T.Type, value: T.Value) -> some View {
```

Comment:
Remove the parameter labels to match the following method, and rename `value` to `newValue`